tools/android/packaging/xbmc/src/channels/SyncProgramsJobService.java.in

```diff
@@ -169,14 +169,6 @@ public class SyncProgramsJobService extends JobService
       {
         if (cursor != null && cursor.moveToNext())
         {
-          Channel channel = Channel.fromCursor(cursor);
-          if (!channel.isBrowsable())
-          {
-            Log.d(TAG, "Channel is not browsable: " + channelId);
-            deletePrograms(channelId, medias);
-          }
-          else
-          {
             XBMCJsonRPC jsonrpc = new XBMCJsonRPC();
```

Comment:
Indentation has to be fixed and closing curly bracket from else block has to be removed too.

Reply:
Yep, see that now.  See if this is better. https://github.com/Tinwarble/xbmc/blob/master/tools/android/packaging/xbmc/src/channels/SyncProgramsJobService.java.in